src/components/ADempiere/DataTable/Windows/index.vue

```diff
@@ -619,9 +619,15 @@ export default defineComponent({
     }
 
     function widthColumn(fieldAttributes) {
-      const { name, display_type } = fieldAttributes
+      const { name, display_type, columnName } = fieldAttributes
       const size = 8
       const lenght = name.length
+      if (columnName === 'C_BPartner_ID' || columnName === 'C_Project_ID' || columnName === 'C_BankAccount_ID') {
```

Comment:
change to ['C_BPartner_ID', 'C_Project_ID', 'C_BankAccount_ID'].includes(columnName)